test/propagation_test.cpp

```diff
@@ -115,7 +115,7 @@ TEST_CASE("SpanContext") {
     }
   }
 
-  SECTION("serialise fails") {
+  SECTION("serialize fails") {
```

Comment:
The original developers were from colonial countries.

Reply:
![](https://wallpaperaccess.com/full/122737.jpg)